GenericsHomework/Node.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace GenericsHomework;
+
+public class Node<T>
+{
+    public T Value { get; set; }
+    public Node<T> Next { get; private set; }
+
+    public Node(T value)
+    {
+        Value = value;
+        Next = this;
+    }
+
+    public override string ToString()
+    {
+            string contents = $"{Value}";
+            return contents;
+    }
+
+    public void Append(T value)
+    {
+        // To add once Exists method is here
+        if (Exists(value)) 
+        {
+            throw new System.ArgumentException("Duplicate Value");
+        }
+         
+
+        Node<T> newNode = new(value);
+        Node<T> cur = this;
+
+        while (cur.Next != this) { cur = cur.Next; }

```

Comment:
This probably should be broken up into multiple lines for readability
```suggestion
        while (cur.Next != this) 
        { 
            cur = cur.Next; 
        }

```

Reply:
Fixed, Good suggestion!